bittensor/synapses/gpt2.py

```diff
@@ -164,7 +160,7 @@ def forward_text(self, inputs: torch.LongTensor):
                 hidden (:obj:`torch.FloatTensor` of shape :obj:`(batch_size, sequence_len, bittensor.__network_dim__)`, `required`): 
                     Hidden layer representation produced using the local_context.
         """
-        hidden = self.forward(inputs=inputs.to(self.device), training = False, remote = False).local_hidden
+        hidden = self.forward(inputs=inputs, training = False, remote = False).local_hidden
```

Comment:
this is the forward call from the server, did you want to take this out? Perhaps it is better to add the to.device() in the nucleus?